src/Place/ReadModel/RDF/RdfProjector.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\ReadModel\RDF;
+
+use Broadway\Domain\DomainMessage;
+use Broadway\EventHandling\EventListener;
+use CultuurNet\UDB3\EventSourcing\ConvertsToGranularEvents;
+use CultuurNet\UDB3\EventSourcing\MainLanguageDefined;
+use CultuurNet\UDB3\Iri\IriGeneratorInterface;
+use CultuurNet\UDB3\Model\ValueObject\Translation\Language;
+use CultuurNet\UDB3\Place\Events\TitleTranslated;
+use CultuurNet\UDB3\Place\Events\TitleUpdated;
+use CultuurNet\UDB3\RDF\GraphRepository;
+use CultuurNet\UDB3\RDF\MainLanguageRepository;
+use EasyRdf\Literal;
+use EasyRdf\Resource;
+
+final class RdfProjector implements EventListener
+{
+    private MainLanguageRepository $mainLanguageRepository;
+    private GraphRepository $graphRepository;
+    private IriGeneratorInterface $iriGenerator;
+
+    private const PROPERTY_LOCATIE_NAAM = 'locn:geographicName';
+
+    public function __construct(
+        MainLanguageRepository $mainLanguageRepository,
+        GraphRepository $graphRepository,
+        IriGeneratorInterface $iriGenerator
+    ) {
+        $this->mainLanguageRepository = $mainLanguageRepository;
+        $this->graphRepository = $graphRepository;
+        $this->iriGenerator = $iriGenerator;
+    }
+
+    public function handle(DomainMessage $domainMessage): void
+    {
+        $payload = $domainMessage->getPayload();
+        $granularEvents = $payload instanceof ConvertsToGranularEvents ? $payload->toGranularEvents() : [];
+        $events = [$payload, ...$granularEvents];
+
+        $uri = $this->iriGenerator->iri($domainMessage->getId());
+
+        $mapping = [
```

Comment:
```suggestion
        $eventClassToHandler = [
```

You could rename it to something like `$eventClassToHandler`. `$mapping` is a bit abstract. But since it's only used here it's clear enough as it is.